clients/python/client/osparc/_solvers_api.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, Iterator, List, Optional, Tuple
+
+from osparc_client import LimitOffsetPageJob
+from osparc_client import SolversApi as _SolversApi
+
+from . import ApiClient, Job
+from ._utils import _pagination_to_iterator
+
+
+class SolversApi(_SolversApi):
+    """Class for interacting with solvers"""
+
+    def __init__(self, api_client: Optional[ApiClient] = None):
+        """Construct object
+
+        Args:
+            api_client (ApiClient, optinal): osparc.ApiClient object
+        """
+        super().__init__(api_client)
+
+    def get_jobs_page(self, solver_key: str, version: str) -> None:
+        """Method only for internal use"""
+        raise NotImplementedError("This method is only for internal use")
+
+    def get_jobs(
+        self, solver_key: str, version: str, limit: int = 20, offset: int = 0
+    ) -> Tuple[Iterator[Job], int]:
```

Comment:
question, why are you using ```Tuple``` vs ```tuple``` is this because you need to be python 3.7 compatible?

Reply:
Yes, we have tests which run with python 3.6